aya/src/maps/mod.rs

```diff
@@ -612,6 +618,16 @@ impl MapData {
         fd
     }
 
+    /// Returns the name of the map
+    pub fn name(&self) -> Result<String, MapError> {
```

Comment:
again let's use the same method as `ProgramInfo.name()` here https://github.com/aya-rs/aya/blob/main/aya/src/programs/mod.rs#L1004

Reply:
Can't quite make it like this yet, as we don't have the struct to reference. Will do as part of follow up PR once we have `MapInfo(bpf_map_info);`.